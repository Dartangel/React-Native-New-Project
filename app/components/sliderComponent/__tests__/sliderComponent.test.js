import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import SliderComponent from '../index';

describe('SliderComponent', () => {
    test('SliderComponent snapshot', () => {
        const renderer = new ShallowRenderer();
        const snapshot = renderer.render(<SliderComponent />);
        expect(snapshot).toMatchSnapshot();
    });
});