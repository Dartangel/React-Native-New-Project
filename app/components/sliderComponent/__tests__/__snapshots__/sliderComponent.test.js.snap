// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SliderComponent SliderComponent snapshot 1`] = `
<View
  style={
    Object {
      "borderRadius": 10,
      "justifyContent": "center",
      "margin": 10,
      "marginLeft": 20,
      "marginRight": 20,
      "width": 320,
    }
  }
>
  <Image
    style={
      Object {
        "borderRadius": 10,
        "height": "100%",
        "width": "100%",
      }
    }
  />
</View>
`;
